### ISSUE TITLE
Can't find model 'fr' despite downloading it

### ISSUE THREAD

**Issue by antoinecomp** [explosion/spaCy#3663]
## How to reproduce the problem

Hi ! I'm trying to install SpaCy in French and I have some issues. Indeed, even if I downloaded the French model with `python -m spacy download fr`, I can't call for as it returns me the following error message:

`OSError: [E050] Can't find model 'fr'. It doesn't seem to be a shortcut link, a Python package or a valid path to a data directory.`

Here is the full error message:

```bash
# copy-paste the error message here
(booenv) C:\Users\antoi\Documents\Programming\PACO\bookchatbot>python -m spacy download fr
Requirement already satisfied: fr_core_news_sm==2.1.0 from https://github.com/explosion/spacy-models/releases/download/fr_core_news_sm-2.1.0/fr_core_news_sm-2.1.0.tar.gz#egg=fr_core_news_sm==2.1.0 in c:\users\antoi\documents\programming\paco\bookchatbot\booenv\lib\site-packages (2.1.0)
You are using pip version 10.0.1, however version 19.1 is available.
You should consider upgrading via the 'python -m pip install --upgrade pip' command.
✔ Download and installation successful
You can now load the model via spacy.load('fr_core_news_sm')
You do not have sufficient privilege to perform this operation.
✔ Linking successful
C:\Users\antoi\Documents\Programming\PACO\bookchatbot\booenv\lib\site-packages\fr_core_news_sm
-->
C:\Users\antoi\Documents\Programming\PACO\bookchatbot\booenv\lib\site-packages\spacy\data\fr
You can now load the model via spacy.load('fr')

(booenv) C:\Users\antoi\Documents\Programming\PACO\bookchatbot>python -m rasa_nlu.train -c nlu_config.yml --data nlu.md -o models --fixed_model_name nlu --project current --verbose
C:\Python37\lib\runpy.py:125: RuntimeWarning: 'rasa_nlu.train' found in sys.modules after import of package 'rasa_nlu', but prior to execution of 'rasa_nlu.train'; this may result in unpredictable behaviour
  warn(RuntimeWarning(msg))
2019-04-30 15:21:55 WARNING  rasa_nlu.config  - You have specified the pipeline template 'spacy_sklearn' which has been renamed to 'pretrained_embeddings_spacy'. Please update your code as it will no longer work with future versions of Rasa NLU.
2019-04-30 15:21:55 INFO     rasa_nlu.utils.spacy_utils  - Trying to load spacy model with name 'fr'
Traceback (most recent call last):
  File "C:\Python37\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "C:\Python37\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\antoi\Documents\Programming\PACO\bookchatbot\booenv\lib\site-packages\rasa_nlu\train.py", line 173, in <module>
    num_threads=cmdline_args.num_threads)
  File "C:\Users\antoi\Documents\Programming\PACO\bookchatbot\booenv\lib\site-packages\rasa_nlu\train.py", line 136, in train
    trainer = Trainer(nlu_config, component_builder)
  File "C:\Users\antoi\Documents\Programming\PACO\bookchatbot\booenv\lib\site-packages\rasa_nlu\model.py", line 150, in __init__
    self.pipeline = self._build_pipeline(cfg, component_builder)
  File "C:\Users\antoi\Documents\Programming\PACO\bookchatbot\booenv\lib\site-packages\rasa_nlu\model.py", line 162, in _build_pipeline
    component = component_builder.create_component(component_cfg, cfg)
  File "C:\Users\antoi\Documents\Programming\PACO\bookchatbot\booenv\lib\site-packages\rasa_nlu\components.py", line 448, in create_component
    component_config, cfg)
  File "C:\Users\antoi\Documents\Programming\PACO\bookchatbot\booenv\lib\site-packages\rasa_nlu\registry.py", line 186, in create_component_by_config
    return component_class.create(component_config, config)
  File "C:\Users\antoi\Documents\Programming\PACO\bookchatbot\booenv\lib\site-packages\rasa_nlu\utils\spacy_utils.py", line 64, in create
    nlp = spacy.load(spacy_model_name, disable=['parser'])
  File "C:\Users\antoi\Documents\Programming\PACO\bookchatbot\booenv\lib\site-packages\spacy\__init__.py", line 27, in load
    return util.load_model(name, **overrides)
  File "C:\Users\antoi\Documents\Programming\PACO\bookchatbot\booenv\lib\site-packages\spacy\util.py", line 136, in load_model
    raise IOError(Errors.E050.format(name=name))
OSError: [E050] Can't find model 'fr'. It doesn't seem to be a shortcut link, a Python package or a valid path to a data directory.
```

I bet it comes from the fact I'm using python 3.7 rather than python 3.6.

I'm doing this in the case of implementing [this Rasa chatbot](https://github.com/antoinecomp/bookchatbot). I changed the model from English to French modifying the [nlu_config.yml file](https://github.com/antoinecomp/bookchatbot/blob/master/nlu_config.yml). 

## Your Environment

* **spaCy version:** 2.1.3
* **Platform:** Windows-10-10.0.17134-SP0
* **Python version:** 3.7.1
* Environment Information:

**Comment by antoinecomp** [comment]
I created a virtual environment with python 3.6 rather than 3.7 and it shifted to that error:

```
(staenv36) C:\Users\antoi\Documents\Programming\Nathalie\18_2_2019\starter-pack-rasa-stack>python -m spacy download fr
Collecting fr_core_news_sm==2.0.0 from https://github.com/explosion/spacy-models/releases/download/fr_core_news_sm-2.0.0/fr_core_news_sm-2.0.0.tar.gz#egg=fr_core_news_sm==2.0.0
  Downloading https://github.com/explosion/spacy-models/releases/download/fr_core_news_sm-2.0.0/fr_core_news_sm-2.0.0.tar.gz (39.8MB)
    100% |████████████████████████████████| 39.8MB 652kB/s
Installing collected packages: fr-core-news-sm
  Running setup.py install for fr-core-news-sm ... done
Successfully installed fr-core-news-sm-2.0.0
You are using pip version 10.0.1, however version 19.1 is available.
You should consider upgrading via the 'python -m pip install --upgrade pip' command.

    Error: Couldn't link model to 'fr'
    Creating a symlink in spacy/data failed. Make sure you have the required
    permissions and try re-running the command as admin, or use a
    virtualenv. You can still import the model as a module and call its
    load() method, or create the symlink manually.

    C:\Users\antoi\Documents\Programming\Nathalie\18_2_2019\starter-pack-rasa-stack\staenv36\lib\site-packages\fr_core_news_sm
    -->
    C:\Users\antoi\Documents\Programming\Nathalie\18_2_2019\starter-pack-rasa-stack\staenv36\lib\site-packages\spacy\data\fr


    Creating a shortcut link for 'en' didn't work (maybe you don't have
    admin permissions?), but you can still load the model via its full
    package name: nlp = spacy.load('{name}')
    Download successful but linking failed
```

**Comment by BramVanroy** [comment]
In both messages, I think it's quite clear what's going on. "Not enough privilege to create shortcut." Run the installation from an elevated prompt. Seems that you are on Windows. Run your command prompt/power shell as Administrator. 

It is true that the final message referring to 'en' is faulty.

**Comment by ines** [comment]
> It is true that the final message referring to 'en' is faulty.

Yes, this was a typo in 2.0.x and should be fixed in 2.1. We also merged a PR that improves the way symlink failures are caught on Windows – so I'm pretty sure in the next update this situation here would cause an error.

If the shortcut isn't super important to you, I'd recommend just using the full model name, `fr_core_news_sm`. That's more straightforward anyways and means spaCy can just load it from the installed package.
